lib/ruby_lsp/requests/support/helpers.rb

```diff
@@ -0,0 +1,25 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  module Requests
+    module Support
+      module Helpers
+        extend T::Sig
+
+        sig { params(signatures: T::Array[RubyIndexer::Entry::Signature]).returns(String) }
+        def format_overloads_count(signatures)
```

Comment:
Any better place to put this?

Reply:
Agreed, this should be in the `Entry::Member` class.